README.md

```diff
@@ -254,6 +259,7 @@ Services to securely store your Docker images.
 
 ## Linter / Validator
 * [dockerfile_lint](https://github.com/projectatomic/dockerfile_lint) - A rule-based 'linter' for Dockerfiles by [@redhataccess](https://github.com/redhataccess)
+* [Hadolint](https://github.com/lukasmartinelli/hadolint) - A Dockerfile linter written in Haskell
```

Comment:
Could you please add the author's GitHub username with respective link after the description? see an example above.